content/setup/proxy_reverse.md

```diff
@@ -45,3 +45,35 @@ location / {
     chunked_transfer_encoding off;
 }
 ```
+
+# Apache2
+
+Example Apache2 reverse proxy configuration:
+
+```
+<VirtualHost *:80>
+
+	ServerName drone.example.com
+	ServerAlias drone
+
+	ServerAdmin webmaster@localhost
+
+	ErrorLog ${APACHE_LOG_DIR}/drone-error.log
+	CustomLog ${APACHE_LOG_DIR}/drone-access.log vhost_combined
+
+	#-#ProxyPass / http://127.0.0.1:8080/
```

Comment:
not needed for sure, but sometimes localhost doesn't resolve same as 127.0.0.1 so left it in.
I'm not attached.